### DISCUSSION TITLE
How do I get console.logs in production on the front-end?

### DISCUSSION THREAD

**Discussion by DanielJackson-Oslo** [vercel/next.js#25372] [Help]
I have a bug that only happens in production, and I get no logs on it which makes it super-hard to debug: 

Any flag for building a production build while keeping console.logs and error messages?

**Comment by Manikanta-20** [comment]
@DanielJackson-Oslo 
If your are facing issue in client side to get those(issue ) in 
log server side for production mode?

**Reply by DanielJackson-Oslo** [reply]
I can see client side logs server side? Where?

**Reply by Manikanta-20** [reply]
yes
can you share the repo link?
